pkg/cmd/show.go

```diff
@@ -360,39 +360,6 @@ func showGrafana(targetReader TargetReader) {
 	checkError(err)
 }
 
-// showKibana shows the kibana dashboard for the targeted cluster
```

Comment:
let me knwo whether if we still need. then I will rollback.

Reply:
should this be really part of this PR?

Reply:
done